protocols/gossipsub/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.45.2 - unreleased
+
```

Comment:
We should document the breaking change here.

Reply:
updated e1bf3fc